src/csharp/BUILD-INTEGRATION.md

```diff
@@ -394,13 +136,13 @@ otherwise unsupported (e.g. experimental) flags to protocol buffer compiler.
 
 * __GrpcOutputOptions__ 
```

Comment:
inconsistency: the Notes section mentions GrpcOutputOptions but doesn't mention OutputOptions - please add it.